app/controllers/maintenance_tasks/tasks_controller.rb

```diff
@@ -18,7 +18,11 @@ def index
     # Renders the page responsible for providing Task actions to users.
     # Shows running and completed instances of the Task.
     def show
-      @task = TaskDataShow.prepare(params.fetch(:id), runs_cursor: params[:cursor])
+      @task = TaskDataShow.prepare(
+        params.fetch(:id),
+        runs_cursor: params[:cursor],
+        arguments: params.except(:id, :cursor, :controller, :action).permit!,
```

Comment:
I just realized this prevents Tasks from having a `cursor` attribute, so I'll remove `:cursor` from this list and the cursor will be used both for the runs and the Task inputs. Which means it may end up showing an empty list of runs below the form to start a task, but I think that's ok.